src/helpers/piping.js

```diff
@@ -0,0 +1,17 @@
+import domready from './domready';
+
+function pipeAnswers() {
+  const pipingDestinations = [...document.querySelectorAll('.js-piped-answer')];
```

Comment:
```suggestion
  const pipingDestinations = [...document.querySelectorAll('.ons-js-piped-answer')];
```

Reply:
@jrbarnes9 We don't need to do this as it's just for the prototype kit.